bos_mint/views.py

```diff
@@ -947,11 +947,13 @@ def genericUpdate(formClass, selectId, removeSubmits=False):
     parentId = None
     if selectId:
         parentId = utils.getParentTypeGetter(typeName)(selectId)
-
+    print("**** Views.py **** selectId, parentId, typeName",\
```

Comment:
Please use logger, don't print directly to console.

Reply:
fixed